index.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+let program = require('caporal');
```

Comment:
It is better to push the caporal dependency and implementation details to lib/controller module. The index.js would only be responsible for configuring and using the objects exported out of lib/controller module.
For our case, the configuration could be just the logger configuration for caporal. The knowledge of utilizing caporal itself is hidden in lib/controller module. The index.js need not know the same.